assets/src/scss/base/_variables.scss

```diff
@@ -10,28 +10,21 @@ $large-width:             992px;
 $extra-large-width:       1200px;
 $extra-extra-large-width: 1600px;
 
-// Units:
-$au: auto;
-$n5: 5px;
-$n10: 10px;
-$n15: 15px;
-$n20: 20px;
-$n25: 25px;
-$n30: 30px;
-$n35: 35px;
-$n40: 40px;
-$n50: 50px;
-$n60: 60px;
-$n70: 70px;
-$n80: 80px;
-$n90: 90px;
-$n100: 100px;
-
 // Spacing
-$space-xxs: 4px;
-$space-xs:  8px;
-$space-sm:  12px;
-$space-md:  16px;
-$space-lg:  32px;
-$space-xl:  48px;
-$space-xxl: 64px;
+$spacer: 8px;
```

Comment:
We can also use numbers and write a CI job that validates them to be multiples of 8.

Reply:
Yeah we could some time on setting up a CI validation, although this feels a bit late in the design process.